### PR TITLE
disable workshops link in onboarding

### PR DESCRIPTION
**Pull request by learykara**
Result:

image

### CODE REVIEW

File: src/AppBar/app-bar.less [hunk 1]
```diff
@@ -186,6 +186,11 @@
     .proxima-bold;
   }
 
+  &.disabled {
+    color: mix(@gray50, @gray25, 40%);
```
Comment (tholex) [inline src/AppBar/app-bar.less]: I think this is `@gray40`

**Comment by tholex** [comment]
:+1:

File: src/AppBar/app-bar.less (hunk 1 above)
Comment (tholex) [inline src/AppBar/app-bar.less]: blue50 or blue75 might work well here

Comment (learykara) [inline src/AppBar/app-bar.less]: oh wow ya that looks way better. I was struggling to make it look not like poop

**Pull request merged**